CollectionGraph/CollectionGraphView.swift

```diff
@@ -11,9 +11,28 @@ import UIKit
 @IBDesignable
 public class CollectionGraphView: UIView {
 
-    @IBInspectable public var myColor: UIColor = UIColor.red {
+    @IBInspectable var layout: UICollectionViewLayout? {
```

Comment:
In a way.  You can add an "object" to the storyboard and assign it the layout class and then link that to the collection view.  That way you can see the inspectable in IB.